pages/search/index.vue

```diff
@@ -214,6 +237,10 @@
     watch: watchList,
     asyncData ({ env, query, res, redirect, app }) {
       const currentPage = pageFromQuery(query.page);
+      const qf = query.qf;
```

Comment:
I don't think any of this belongs here. The tier toggle is already in qfForSelectedFacets if it's present, which is also then wired up with updateCurrentSearchQuery to update the search results when the qf changes and keep the values of the qfs in selectedFacets.